### ISSUE TITLE
Feature Request: Add animation ease curves to key frame animations

### ISSUE THREAD

**Issue by AndreMuis** [Microsoft/mixed-reality-extension-sdk#167]
Add the ability to specify the animation ease curve for a key frame animation. Currently it looks like the animations are hard-coded to ease-in / ease-out. 

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/920149/52180836-ec583d80-279f-11e9-9f66-4666e18e09f2.gif)

[ease-curve.ts.txt](https://github.com/Microsoft/mixed-reality-extension-sdk/files/2825859/ease-curve.ts.txt)

**Comment by eanders-MS** [comment]
I think we could do this with a helper function to generate some keyframes between two points on a curve.

**Comment by stevenvergenz** [comment]
Currently keyframe animations are linearly interpolated, aren't they @eanders-MS ?

**Comment by eanders-MS** [comment]
Yep, currently linear. But it would be straight forward to add an ease curve parameter to the keyframe.

**Comment by AndreMuis** [comment]
That animation slows down at the top and bottom unless I'm missing something.

**Comment by stevenvergenz** [comment]
It does look like it's easing, yeah...

**Comment by eanders-MS** [comment]
@stevenvergenz  what is the bug here?
